docs/deprecated/rancher.md

```diff
@@ -127,8 +127,8 @@ kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/relea
 Add a label to `Namespace` resources for the TopoLVM webhook to avoid unnecessary validation.
 
 ```console
-kubectl label namespace kube-system topolvm.cybozu.com/webhook=ignore
```

Comment:
These changes are not needed, because this document is currently not maintained.